README.md

```diff
@@ -21,7 +22,11 @@ Can be done without pulling this app into a ServiceNow instance
 
 If approved, your new parser automatically goes live for the bot's immediate use!
 
-### Regular method (Adding new functionality types)
+<br>
+
+### **Hard method (Adding new functionality types)**
```

Comment:
this breaks the anchor link on line 12, make sure to edit the anchor link to match this new header3 name